daemon/config_unix.go

```diff
@@ -82,6 +85,37 @@ func (config *Config) InstallFlags(cmd *flag.FlagSet, usageFn func(string) strin
 	cmd.StringVar(&config.CgroupParent, []string{"-cgroup-parent"}, "", usageFn("Set parent cgroup for all containers"))
 	cmd.StringVar(&config.RemappedRoot, []string{"-userns-remap"}, "", usageFn("User/Group setting for user namespaces"))
 	cmd.StringVar(&config.ContainerdAddr, []string{"-containerd"}, "", usageFn("Path to containerd socket"))
+	config.Runtimes = make(map[string]runtime.Runtime)
+	cmd.Var(runconfigopts.NewNamedRuntimeOpt("runtimes", &config.Runtimes), []string{"-runtime"}, usageFn("Add an OCI runtime to the list of available runtimes"))
```

Comment:
I vote to `--add-runtime`, because containerd has a `--runtime` flag with different meaning :smile: